.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs :
       # solution.
       fail-fast : false
       matrix :
-        kotlin-version : [ 1.8.0 ]
+        kotlin-version : [ 1.8.20-Beta ]
```

Comment:
There are a couple other jobs that also have this matrix as well like `test-ubuntu`